src/translator/text-based-field.ts

```diff
@@ -123,6 +120,48 @@ export namespace TextBasedField {
         return validationObj;
     };
 
+    export const parseTextBasedFieldValidation = (
```

Comment:
ideally we want to:
- differentiate between element types 
- aside from checking for the presence of the key, check if it is true 
but I will let these changes get through first since they will be refactored